target/linux/ar71xx/image/legacy-devices.mk

```diff
@@ -427,6 +427,12 @@ define LegacyDevice/dLAN_pro_1200_ac
 endef
 LEGACY_DEVICES += dLAN_pro_1200_ac
 
+define LegacyDevice/DW33D
+  DEVICE_TITLE := DomyWifi DW33D
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ledtrig-usbdev kmod-ath10k
+endef
+LEGACY_DEVICES += DW33D
```

Comment:
> It's fixed. But build the image size is only 4.7M.

4.7 MByte sounds pretty normal to me. What was the image size using the old build code? What is the problem with an image of only 4.7 MByte?

>  Error message：The uploaded image file does not contain a supported format. 

Is this error message new or do you had the same error message using the old build code? Where does this error message come from? My first guess is that it's a message from the update webpage of the stock firmware. In that case, you might need to add a factory image to the build code (one with a special header or similar).